packages/dds/tree/src/core/tree/cursor.ts

```diff
@@ -9,11 +9,28 @@ import { FieldKey, TreeType, Value } from "./types";
 
 /**
  * A stateful low-level interface for reading tree data.
+ *
+ * @remarks Cursors exist so that specialized implementations can be written against different data abstractions.
+ * This allows performance optimizations to be done based on a data.
  */
 export interface ITreeCursor {
     /**
      * What kind of place the cursor is at.
      * Determines which operations are allowed.
+     *
+     * @remarks Users of cursors frequently need to refer to places in trees, both fields and nodes.
```

Comment:
I thought paragraph breaks ended tags like this. Is that not the case? Is there an easy way to tell what content ends up in which sections of the docs?

Reply:
My understanding is that everything is part of its parent tag until the next tag starts. I'm not sure though and it's not currently easy to verify because our currently published docs don't seem to be complicated enough to have run into this issue yet. I can verify once all the internal stuff has been published.

Reply:
Looking at https://api-extractor.com/pages/tsdoc/tag_remarks/ I think you are correct. Given that, I think its better to put `@remarks` on its own line like those examples do to make this more clear that this is a separator between sections and not a modifier for this paragraph or line.
```suggestion
     * @remarks
     * Users of cursors frequently need to refer to places in trees, both fields and nodes.
```

https://tsdoc.org/pages/spec/tag_kinds/ clarifies that this tag should be on its own line in "normalized form"